gqlalchemy/query_builder.py

```diff
@@ -1120,3 +1120,9 @@ def __init__(
     ):
         super().__init__(connection)
         self._query.append(WithPartialQuery(results))
+
+
+class LoadCSV(DeclarativeBase):
+    def __init__(self, path: str, header: bool, row: str, connection: Optional[Union[Connection, Memgraph]] = None):
+        super().__init__(connection)
+        self._query.append(LoadCsvPartialQuery(path, header, row))
```

Comment:
We have LoadCSV above and then LoadCsvPartialQuery. We should be more consistent and decide on one of those two. I vote for LoadCSVPartialQuery 😄